stdlib/public/core/StringCreate.swift

```diff
@@ -211,25 +211,57 @@ extension String {
       return unsafe repairUTF8(result.codeUnits, firstKnownBrokenRange: initialRange)
     }
   }
+  
+  @usableFromInline
+  internal static func _uncheckedFromUTF8(
+    _ input: UnsafeBufferPointer<UInt8>,
+  ) -> String {
+    return unsafe _uncheckedFromUTF8(
+      input,
+      isASCII: _allASCII(input),
+      precalculatedUTF16Count: nil
+    )
+  }
 
   @usableFromInline
   internal static func _uncheckedFromUTF8(
-    _ input: UnsafeBufferPointer<UInt8>
+    _ input: UnsafeBufferPointer<UInt8>,
```

Comment:
This is an overload purely because the old version is exposed as ABI due to usableFromInline